libc/stdlib.c

```diff
@@ -53,7 +53,7 @@ void exit(int status) {
         GetKey(&key);
 }
 
-void abort() {
+void abort(void) {
```

Comment:
This one is unnecessary because the function prototype is in the header, right?

Reply:
I'm not sure. Prior to creating the pull request I tested and it compiled. I know it's possible to receive errors or warnings because the implementation and the prototype don't match. I did this so the prototype and the implementation match. I think worst case this change won't harm or change anything.